src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
+  gap: $card-spacing;
+  max-width: $card-width * 4 + $card-spacing * 3;
   margin: 0 auto;
 
+  @media (max-width: 488px) {
+    grid-template-columns: $card-width;
+  }
```

Comment:
This media query is not needed, just write such styles by default, other media queries will overlap